Tests/Rules/UseProcessBlockForPipelineCommand.tests.ps1

```diff
@@ -0,0 +1,37 @@
+﻿Describe "UseProcessBlockForPipelineCommand" {
+    BeforeAll {
+        $RuleName = 'PSUseProcessBlockForPipelineCommand'
+        $NoProcessBlock = 'function BadFunc1 { [CmdletBinding()] param ([Parameter(ValueFromPipeline)]$Param1) }'
+        $NoProcessBlockByPropertyName = 'function $BadFunc2 { [CmdletBinding()] param ([Parameter(ValueFromPipelineByPropertyName)]$Param1) }'
+        $HasProcessBlock = 'function GoodFunc1 { [CmdletBinding()] param ([Parameter(ValueFromPipeline)]$Param1) process { } }'
+        $HasProcessBlockByPropertyName = 'function GoodFunc2 { [CmdletBinding()] param ([Parameter(ValueFromPipelineByPropertyName)]$Param1) process { } }'
+        $NoAttribute = 'function GoodFunc3 { [CmdletBinding()] param ([Parameter()]$Param1) }'
+		$HasTypeDeclaration = 'function GoodFunc3 { [CmdletBinding()] param ([Parameter()][string]$Param1) }'
+    }
+
+    Context "When there are violations" {
+        $Cases = @(
```

Comment:
Please move this to the `BeforeAll` or to be a literal value in the `TestCases` parameter

Reply:
fixed in #707f02d96ac5763afaf0b69649d787f8f25b2492